ember/app/router.js

```diff
@@ -9,13 +9,18 @@ export default class Router extends EmberRouter {
 
 //eslint-disable-next-line array-callback-return
 Router.map(function () {
-  this.mount("ember-emeis", { as: "emeis", path: "/emeis" });
-  this.mount("ember-alexandria", { as: "alexandria", path: "/alexandria" });
-  this.mount("ember-caluma", { as: "form-builder", path: "/form-builder" });
-  this.route("cases", { path: "/" }, function () {
-    this.route("detail", { path: "/:id" }, function () {
-      this.route("edit");
+  this.route("login");
+  this.route("logout");
+
+  this.route("protected", { path: "/" }, function () {
```

Comment:
```suggestion
  this.route("protected", { path: "/", resetNamespace: true }, function () {
```

So you don't have to prefix every link with `protected.`

Reply:
Done.